src/app/src/main/kotlin/org/vaccineimpact/api/app/serialization/Serializer.kt

```diff
@@ -79,6 +80,13 @@ open class Serializer
         return text.toLowerCase().replace('_', '-')
     }
 
+    fun serializeValue(value: Any?) = when (value)
```

Comment:
Maybe in this context it needs a more explicit name? e.g. `serializeValueForCSV`